cypress/src/tests/e2eTests/verifyProductSearch.spec.js

```diff
@@ -0,0 +1,95 @@
+
+import {
+  assertImageListDisplay,
+  assertSearchResultClick,
+  aseertSearchResults
+} from "../../assertions";
+
+import {
+  inputSearchString
+} from "../../actions";
+
+import * as fields from "../../fields";
+
+describe("Search Feature", () => {
+  it("Verify quick search features", () => {
+    // Visit the homepage
+    cy.visit("/");
+
+    //Input search string
+    inputSearchString("tops");
+
+    aseertSearchResults();
+
+    assertImageListDisplay('.product-discovery-product-list__grid');
+
+    assertSearchResultClick();
+
+  });
+
+  it("Verify Search results page", () => {
+    // Visit the homepage
+    cy.visit("/");
+
+    // Input search string and hit enter
+    inputSearchString("sleeve{enter}");
+
+    // Wait for random quick search dropdown to disappear
+    cy.wait(1000);
+
+    aseertSearchResults();
```

Comment:
should be spelled `assertSearchResults`
```suggestion
export const assertSearchResults = () => {
```